config/locales/en.yml

```
@@ -1906,6 +1909,8 @@ en:
     upsale:
       title: "Enterprise add-on"
       link_title: "More information"
+      customize_life_cycle:
+        description: "Create and organise different project stages and gates than the ones provided by PM2 project cycle planning."
```

Using American English:

```suggestion
        description: "Create and organize different project stages and gates than the ones provided by PM2 project cycle planning."
```